server/router/routes/plan.js

```diff
@@ -172,14 +180,35 @@ router.put('/:planId?', asyncMiddleware(async (req, res) => {
   }
 }));
 
+const updatePlanStatus = async (planId, status = {}, user) => {
+  const plan = await Plan.findOne(db, { id: planId });
```

Comment:
I'd put the the await in a try catch for good practices. It's hard to track down an unhandled promise warning.